web/src/app/modules/sugarloaf/components/smart/container/container.component.html

```diff
@@ -15,6 +15,10 @@
       </div>
     </div>
     <div class="header-actions">
+      <a [routerLink]="['/configuration/upload']" class="header-link">
+        <clr-icon shape="upload"></clr-icon>
+        <span>Upload</span>
```

Comment:
Same here, lets call it `Apply YAML` and keep the upload arrow. I think that just the word upload is too vauge.